pkg/networkserver/downlink.go

```diff
@@ -952,7 +962,12 @@ func (ns *NetworkServer) scheduleDownlinkByPaths(ctx context.Context, req *sched
 			errs = append(errs, err)
 			continue
 		}
-		transmitAt := time.Now().Add(delay)
+		var transmitAt time.Time
+		if absTime := req.AbsoluteTime; absTime != nil {
```

Comment:
Adding the `delay` to `time.Now` is not taking into account the round trip times. For arbitrary time scheduling this is acceptable, but for absolute time scheduling we can be more precise.